src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -271,8 +271,9 @@ public IEnumerable<PackageItemListViewModel> GetCurrent()
                         Versions = AsyncLazy.New(() => metadata.GetVersionsAsync()),
                         AllowedVersions = allowedVersions,
                         PrefixReserved = metadata.PrefixReserved && !IsMultiSource,
-                        IsPackageDeprecated = metadata.DeprecationMetadata != null,
+                        DeprecationMetadata = AsyncLazy.New(() => metadata.GetDeprecationMetadataAsync()),
```

Comment:
I think you don't need the extraneous lambda:

    DeprecationMetadata = AsyncLazy.New(metadata.GetDeprecationMetadata);

Reply:
yup.